Fix failing github actions

After #27, the `flake8` linting step is [failing](https://github.com/NOAA-OWP/hypy/actions/runs/5446331278/jobs/9906907888#step:5:18). 

```shell
./python/hypy/hydrolocation/nwis_location.py:17:30: F821 undefined name 'Point'
                 shape: Union['Point', Tuple] = None,
                              ^
 1     F821 undefined name 'Point'
 1
 ##[error]Process completed with exit code 1.
```

--- a/python/hypy/hydrolocation/nwis_location.py
+++ b/python/hypy/hydrolocation/nwis_location.py
@@ -7,6 +7,7 @@ if TYPE_CHECKING:
     from pandas import DataFrame, Timestamp
     from numpy import datetime64
     from datetime import datetime
+    from shapely.geometry import Point
 class NWISLocation(HydroLocation):
     """
         An NWIS subclass of HydroLocation

